lib/t/feeds.ex

```diff
@@ -45,9 +45,15 @@ defmodule T.Feeds do
 
     for user_id <- user_ids do
       broadcast(topic, {__MODULE__, :deactivated, user_id})
+      schedule_deactivated_push_notification(user_id)
```

Comment:
`notify_deactivated` are only for in-app broadcasts

Reply:
yeah it was. does adding push_scheduling somehow breaks it / slows it down?

Reply:
yes, it needs to stay the way it was

Reply:
could you please explain the logic?  (I'd like to understand who it works)

Reply:
that's just how it works right now

Reply:
I guess it's usually called separation of concerns